package.json

```diff
@@ -5,11 +5,15 @@
   "description": "Frontend build tools and development utilities for Kolibri Studio",
   "scripts": {
     "lint:py": "flake8 --exit-zero",
+    "lint-frontend": "kolibri-tools lint --pattern 'contentcuration/contentcuration/frontend/**/*.{js,vue,scss,less,css}' --ignore '**/dist/**,**/node_modules/**,**/static/**'",
+    "lint-frontend:format": "yarn run lint-frontend --write",
+    "lint-frontend:watch": "yarn run lint-frontend --monitor",
+    "lint-frontend:watch:format": "yarn run lint-frontend --monitor --write",
     "lint": "kolibri-tools lint",
-    "lint-all": "yarn run lint -p 'contentcuration/contentcuration/frontend/**/*.{js,vue,scss,less,css}'",
-    "lint-all:fix": "yarn run lint-all -w",
+    "lint-all": "yarn run lint-frontend",
+    "lint-all:fix": "yarn run lint-frontend:format",
     "lint:fix": "yarn run lint -w",
-    "lint:watch": "yarn lint-all -w -m",
+    "lint:watch": "yarn lint-frontend:watch:format",
```

Comment:
Should we just delete this? And others that are just aliases

Reply:
Yeah when I've run linting, I just went straight to the source, e.g. `npx kolibri-tools lint ...`. I suspect this will be fine as long as we announce the change

Reply:
Announced! And updated. I searched the codebase for any residual references and think I've updated them all.